R/filterFeatures.R

```diff
@@ -163,12 +161,17 @@ filterFeatures = function(task, method = "randomForestSRC_importance", fval = NU
       stopf("You supplied multiple filters. Please choose which should be used for the final subsetting of the features.")
     }
     if (is.null(select.method)) {
-      fval = fval[fval$method == fval$method, ]
+      fval = fval[fval$method == method[[1]], ]
```

Comment:
This looks dangerous?

Reply:
Perhaps. But do you agree that 
`fval = fval[fval$method == fval$method, ]`
has no effect?